views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,11 @@
+  <div id="container-beer">
+    <p>Nombre:{{this.name}}</p>
```

Comment:
Cuando utilizamos this estamos haciendo referencia al mismo objeto, entonces puedes utilizarlo o no.
Yo recomiendo omitirlo para que no te confundas y tu código se vea mas limpio aun. 👍